fenix/app/src/main/java/org/mozilla/fenix/downloads/DynamicDownloadDialog.kt

```diff
@@ -74,6 +76,11 @@ class DynamicDownloadDialog(
                     mozilla.components.feature.downloads.R.string.mozac_feature_downloads_button_try_again,
                 )
                 setOnClickListener {
+                    context.components.core.store.dispatch(
```

Comment:
We should probably fix this in A-C. Looking at the code it looks like there is an [`ACTION_TRY_AGAIN`](https://github.com/mozilla-mobile/firefox-android/blob/main/android-components/components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt#LL207C21-L207C37) action that we may be missing in the [intent that we're starting](https://github.com/mozilla-mobile/firefox-android/blob/main/android-components/components/feature/downloads/src/main/java/mozilla/components/feature/downloads/manager/FetchDownloadManager.kt#L89).

Does restarting the download using the [notification](https://github.com/mozilla-mobile/firefox-android/blob/main/android-components/components/feature/downloads/src/main/java/mozilla/components/feature/downloads/DownloadNotification.kt#L167) work? It looks like the pending intent there is passing the `ACTION_TRY_AGAIN` action

Reply:
Hi, yes from notification is working. I did the changes in A-C.